syntax = "proto2";

message request_msg2
{
	required string info = 1;
}